deps/patches/libgit2-bundled_zlib.patch

```diff
@@ -1,55 +0,0 @@
-From 637f75b2a341df6e308f8a20827dea938a39bd35 Mon Sep 17 00:00:00 2001
-From: Valentin Churavy <v.churavy@gmail.com>
```

Comment:
@vchuravy From what I can tell, the zlib issue now appears to now be fixed upstream (https://github.com/libgit2/libgit2/pull/4403). Could you check this?